specification/security/resource-manager/Microsoft.Security/preview/2017-08-01-preview/examples/AutoProvisioningSettings/CreateAutoProvisioningSettingsSubscription_example.json

```diff
@@ -8,7 +8,7 @@
             "name": "default",
             "type": "Microsoft.Security/autoProvisioningSettings",
             "properties": {
-                "provisioningSetting": "Automatic"

```

Comment:
nit: `_example` for the file name unnecessarily increases the file path size, you could simply rename this as `CreateAutoProvisioningSettingsSubscription.json`
not a blocker

Reply:
we have simultaneous PRs on this repo, in order to avoid conflicts we will change it after they will be completed 😄